ast_release.md

```diff
@@ -0,0 +1,48 @@
+# Deploying the OQ3 Python AST
+
+Author: Jake Lishman `jake.lishman@ibm.com`
```

Comment:
Thanks for thinking of me, but no particular need to credit my quickly typed Slack message!  If you wanted to for history purposes, you can add me as a co-author to the commit that creates this file (I use `Co-authored-by: Jake Lishman <jake.lishman@ibm.com>` for work-related commits), but otherwise no worries.

If we write it in, it'll just stale as people edit things in the future.